metaflow/cli_components/run_cmds.py

```diff
@@ -380,3 +395,126 @@ def run(
             )
     with runtime.run_heartbeat():
         runtime.execute()
+
+
+@click.command(help="Spins up a task for a given step from a previous run locally.")
+@click.option(
```

Comment:
I would make this an argument. It feels maybe a little more natural to say: `myflow.py spin my_step` than `myflow.py spin --step-name my_step` It would also simplify things a bit as these are either/or option with pathspec.